Test/TestConfigurationUtils.ps1

```diff
@@ -108,31 +109,22 @@ function Test-IsVRouterExtensionEnabled {
 
 function Start-DockerDriver {
     Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session,
-           [Parameter(Mandatory = $true)] [string] $AdapterName,
-           [Parameter(Mandatory = $true)] [OpenStackConfig] $OpenStackConfig,
-           [Parameter(Mandatory = $true)] [ControllerConfig] $ControllerConfig,
            [Parameter(Mandatory = $false)] [int] $WaitTime = 60)
-
     Write-Log "Starting Docker Driver"
 
     # We have to specify some file, because docker driver doesn't
     # currently support stderr-only logging.
     # TODO: Remove this when after "no log file" option is supported.
     $OldLogPath = "NUL"
-
     $LogDir = Get-ComputeLogsDir
+    $DefaultConfigFilePath = Get-DefaultCNMPluginsConfigPath
 
+    # TODO: delete the "config" argument
```

Comment:
I'm assuming you saw this: https://github.com/Juniper/contrail-windows-docker-driver/blob/master/configuration/configuration.go#L60-L64 :) 
```
func DefaultConfigFilepath() string {
	return ""
	// TODO: uncomment the following lines once templating logic is implemented in ansible.
	//       For now, return empty string, so that docker driver defaults to reading only command
	//       line parameters.
	// return string(filepath.Join(os.Getenv("ProgramData"),
	// 	"Contrail", "etc", "contrail", "cnm-driver.conf"))
}
```
Yup, it should be done